plugins/output/hdfs/mock_hdfs_output_test.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+/**
+* Copyright 2016 Confluent Inc.
```

Comment:
Add this copyright notice to our LICENSES file since we've included it directly in our project